app/messages/service_route_binding_show_message.rb

```diff
@@ -0,0 +1,17 @@
+require 'messages/list_message'
+
+module VCAP
+  module CloudController
+    class ServiceRouteBindingShowMessage < BaseMessage
+      QUERY_PARAMS = %w[
+        include
+      ].freeze
+
+      register_allowed_keys QUERY_PARAMS.map(&:to_sym)
```

Comment:
The message is missing the validation the values allowed for `include`. The same for the list message as well